Code/Framework/AzToolsFramework/AzToolsFramework/UI/Logging/LogLine.cpp

```diff
@@ -405,7 +405,7 @@ namespace AzToolsFramework
                 severity = "Context";
                 break;
             }
-            return dateTime + separator + severity + separator + m_message.c_str() + From + m_window.c_str() + EOL;
+            return dateTime + separator + severity + separator + Origin + m_window.c_str() + separator + m_message.c_str() + EOL;
```

Comment:
minor: the case or Origin is inconsistent with separator, severity, m_window, m_message. I'd recommend making it `origin`